src/components/Navigation/styled.js

```diff
@@ -23,64 +28,81 @@ export const Container = styled.nav`
 
 export const Wrapper = styled.div`
   display: flex;
-  align-items: center;
+  height: 100%;
+  flex-wrap: wrap;
   justify-content: space-between;
-  gap: 80px;
-  max-width: 1368px;
-  padding: 23px 16px 23px 17px;
-  margin: 0 auto;
   align-items: center;
+  max-width: 1416px;
+  padding-left: 40px ;
+  padding-right: 41px;
+  margin: 0 auto;
   transition: 0.6s;
 
-  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
-    padding: 24px 16px 16px 16px;
-    flex-wrap: wrap;
+  @media (max-width: 1080px) {
     justify-content: center;
-    gap: 24px;
+    align-items: flex-start;
+    height: 142px;
+    align-content: flex-start;
   }
 
+  @media (max-width: ${({ theme }) => theme.breakpoint.medium}) {
+    padding-left: clamp(16px, 3vw, 40px);
+    padding-right: clamp(17px, 3vw, 41px);
+  }
+  
   @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
-    gap: 19px;
+    padding-left: 16px;
+    padding-right: 17px;
   }
 `;
 
 export const LogoButtonsWrapper = styled.div`
   display: flex;
-  gap: 80px;
-  flex-shrink: 1;
-  transition: 0.6s;
-
-  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
-    gap: 40px;
-  }
+  height: 34px;
+  margin: 23px 0;
+  align-items: center;
 
-  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
-    gap: 19px;
+  @media (max-width: 1080px) {
+    justify-content: center;
+    margin: 24px 0; 
+    width: 100%;
+    justify-content: space-between;
   }
 `;
 
-export const Logo = styled.div`
+export const AppName = styled.div`
   display: flex;
   align-items: center;
-  justify-content: center;
-  flex-shrink: 0;
   gap: 12px;
   font-weight: ${({ theme }) => theme.fontWeight.medium};
   font-size: 24px;
   line-height: 40px;
+  letter-spacing: -1.5px;
+  text-align: left;
+  padding-right: 80px ;
+  min-width: max-content;
   transition: 0.6s;
+  
+  @media (max-width: ${({ theme }) => theme.breakpoint.medium}) {
+    padding-right: 19px ;
+    font-size: clamp(13px, 3vw, 13px);
```

Comment:
I would remove this line of code. With this font reduction, the app name appears very small on screens between 480px and 767px. In the first photo below there is a difference when this line of code is present and in the second one when we remove it.
1.
![1](https://github.com/Urbork/movies-browser/assets/129622893/86d1aa3e-3c3f-4fba-b8d8-b5e6cdf1d69a)

2.
![2](https://github.com/Urbork/movies-browser/assets/129622893/7c567d55-cd78-4322-8996-dc2468e7ee40)

Reply:
Removed unnecessary font-size from AppName. 
It really is much better now.